addons/notes/src/addon.ts

```diff
@@ -1,8 +1,10 @@
 import addons, { makeDecorator } from '@storybook/addons';
-import marked from 'marked';
+import { MarkedOptions, parse as marked } from 'marked';
 
-function renderMarkdown(text, options) {
-  return marked(text, { ...marked.defaults, ...options });
+// todo why not just calling marked directly instead of wrapping it?
+// todo removed { marked.defaults, options } merge. I believe this was not necessary, have to check
```

Comment:
There is no need in the wrapper or merging options. You use exposed `parse` attr.

[parse is actually an alias](https://github.com/markedjs/marked/blob/f8d556a1cbcab1e5c4eb0acea066316b48a9403c/lib/marked.js#L1610) and inside of `parse` it is [merging options](https://github.com/markedjs/marked/blob/f8d556a1cbcab1e5c4eb0acea066316b48a9403c/lib/marked.js#L1547)

So you can write just `import { parse as renderMarkdown } from 'marked';`

Reply:
Done